packages/round-manager/src/features/round/ViewRoundResults/ViewRoundResults.tsx

```diff
@@ -153,6 +163,36 @@ export default function ViewRoundResults() {
   const [distributionOption, setDistributionOption] = useState<
     "keep" | "scale"
   >("keep");
+  const [roundSaturation, setRoundSaturation] = useState<number>(0);
+  const [sumTotalMatch, setSumTotalMatch] = useState<number>(0);
+
+  const mutateMatchingFundsCallback = useCallback(mutateMatchingFunds, [
+    mutateMatchingFunds,
+  ]);
+
+  useEffect(() => {
+    mutateMatchingFundsCallback();
+    if (round && matches) {
+      const sumTotalMatch = matches?.reduce(
+        (acc: number, match) =>
+          acc +
+          (areMatchingFundsRevised
+            ? Number(match.revisedMatch) / 10 ** 18
```

Comment:
revisedMatch is not guaranteed to fit into a number

Reply:
we should use token amounts exclusively, you can use decimal places to represent decimal percentages